src/app/modules/recipe-list/components/recipe-carousel.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, HostListener, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-recipe-carousel',
+  templateUrl: './recipe-carousel.component.html',
+  styleUrls: ['./recipe-carousel.component.scss']
+})
+export class RecipeCarouselComponent {
+
+  public limit = 3;
+  images = [
```

Comment:
use always public
```suggestion
  public limit = 3;
  public images = [
```
or never do that (make your code consistent)
```suggestion
  limit = 3;
  images = [
```